.github/workflows/check-installation.yml

```diff
@@ -20,3 +20,4 @@ jobs:
           }
           IEX (IWR 'https://raw.githubusercontent.com/${{ github.event.pull_request.head.repo.full_name }}/${{ github.event.pull_request.head.ref }}/install-atomicsfolder.ps1' -UseBasicParsing);
           Install-AtomicsFolder -Force
+          Invoke-AtomicTest All -ShowDetailsBrief
```

Comment:
Running Invoke-AtomicTest to check whether the commands actually work after installing AtomicRedTeam